.travis.yml

```diff
@@ -95,6 +87,9 @@ jobs:
       python: 3.8
       script: coverage run $(which nosetests) -c nose_unit.cfg
 
+      # This is not necessarily going to be forever -- once we merge yt-4.0
```

Comment:
hmmm this seems out of date?

Reply:
That was copied from current master :) For the sake of future merge conflicts I'd leave it without any changes